docs/design-api_guidance.html

```
@@ -194,22 +503,22 @@ <h1>API Guidance</h1>
 <li><strong>Evolve Independently from Clients</strong>: Versioning of the API to allow clients to continue to use the API in a previous state</li>
 <li><strong>Resource Based</strong>: Every API should be designed around a resource (model representation of an object)</li>
 </ul>
-<h1 id="api-standardization">API Standardization <a id="api-standardization" href="#api-standardization" class="markdown-heading-anchor text-decoration-none" aria-label="Link to section: API Standardization"><i class="bi bi-link-45deg" style="opacity: 0.2; font-size: 20px;"></i></a></h1>
-<h2 id="swagger">Swagger <a id="swagger" href="#swagger" class="markdown-heading-anchor text-decoration-none" aria-label="Link to section: Swagger"><i class="bi bi-link-45deg" style="opacity: 0.2; font-size: 20px;"></i></a></h2>
+<h1 id="api-standardization" class="content-heading">API Standardization <a id="api-standardization" href="#api-standardization" class="markdown-heading-anchor text-decoration-none" aria-label="Link to section: API Standardization"><i class="bi bi-link-45deg" style="opacity: 0.2; font-size: 20px;"></i></a></h1>
```

HTMLHint (1.9.2)

[error] The id value [ api-standardization ] must be unique.

(id-unique)